### PR TITLE
[Rule Tuning] Command Shell Activity Started via RunDLL32

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/995

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by brokensound77 — APPROVED** [review]
bump `updated_date` and LGTM 👍

File: rules/windows/execution_command_shell_via_rundll32.toml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 [metadata]
 creation_date = "2020/10/19"
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2021/03/08"
 
 [rule]
 author = ["Elastic"]
 description = "Identifies command shell activity started via RunDLL32, which is commonly abused by attackers to host malicious code."
+false_positives = ["Microsoft Windows Installers Leveraging Rundll32 for installation."]
```
Comment (bm11100) [inline rules/windows/execution_command_shell_via_rundll32.toml:9]:
```suggestion
false_positives = ["Microsoft Windows installers leveraging RunDLL32 for installation."]
```

File: rules/windows/execution_command_shell_via_rundll32.toml [hunk 2]
```diff
@@ -21,9 +22,10 @@ type = "eql"
 query = '''
 process where event.type in ("start", "process_started") and
```
Comment (bm11100) [inline rules/windows/execution_command_shell_via_rundll32.toml:23]:
```suggestion
process where event.type == "start" and
```

**Review by bm11100 — APPROVED** [review]
🚅

**Pull request merged**